pallets/bitcoin_locks/src/lib.rs

```diff
@@ -886,18 +886,25 @@ pub mod pallet {
 			}
 
 			// burn the current redemption price from the vault
-			let redemption_rate =
-				Self::get_redemption_price(&lock.satoshis, Some(lock.lock_price))?;
+			// Choose rate based on whether this was external theft or expiration
+			let burn_rate = if is_externally_spent {
+				// External spend = theft/collusion, penalize at market value
+				T::PriceProvider::get_bitcoin_argon_price(lock.satoshis)
+					.ok_or(Error::<T>::NoBitcoinPricesAvailable)?
+			} else {
+				// Lock/UTXO expiration = user didn't redeem, use redemption rate with penalty curve
+				Self::get_redemption_price(&lock.satoshis, Some(lock.lock_price))?
+			};
```

Comment:
This doesn't seem right. It seems the redemption rate should be the same regardless.

Reply:
Additionally, "externally spent" requires some mental effort to comprehend its meaning. Should we just call it "theft"?